### PR TITLE
F#20 pictures

### LINKED ISSUE #20: Models of pictures in IT-space
Upload models of pictures in IT-space.
blend file of frame.
dae and sdf files of each picture.
Name of pictures would be p(number of picture from left to right)_(surname of person in photo).

### PR DESCRIPTION
**Pull request by sofiia-tesliuk**
close #20

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -81,4 +81,7 @@ qtcreator-*
 .idea
 
 # FreeCAD backup files
-*.fcstd1
\ No newline at end of file
+*.fcstd1
+
+.DS_Store
+robert_gazebo/meshes/.DS_Store
```
Comment (AndriyPt) [inline .gitignore]: these is no need to add .DS_Store ignore in each folder. Rule in row no 86 will ignore it in every folder.

**Review by AndriyPt — CHANGES_REQUESTED** [review]
I would advise to use Leobot approach e.g. https://github.com/ros-ukraine/leobot/tree/kinetic-devel/leobot_gazebo/models/room1_img3 which do not require dae files only JPG :)

**Comment by sofiia-tesliuk** [comment]
Done.

**Review by AndriyPt — CHANGES_REQUESTED** [review]
Please remove debug code from ucu.world and optionally add inertia to pictures.

**Comment by sophia-tslk** [comment]
Done

**Comment by sofiia-tesliuk** [comment]
Done

**Review by AndriyPt — APPROVED** [review]

**Pull request merged**